### PR TITLE
[Test] Replace unittest ksonnet code with python based

### LINKED ISSUE #430: Get rid of ksonnet in the tests
ksonnet was deprecated over a year ago the tests should no longer be using ksonnet to define tests.

Related to #774

### PR DESCRIPTION
**Pull request by PatrickXYS**
Resolves #430

As part of removing ksonnet work, I replaced existing unittest which is written by ksonnet with python unit test.

/hold

Hold until the tests passes

### CODE REVIEW

**Comment by PatrickXYS** [comment]
Failed for `make: *** No rule to make target 'go-unittest-junit'.  Stop.`

Fix for restarting

**Comment by PatrickXYS** [comment]
Confirmed tests passed, clean up left code.

**Comment by PatrickXYS** [comment]
@theofpa Can you help review the code?

File: Makefile [hunk 1]
```diff
@@ -303,6 +303,12 @@ test-junit: build-kfctl
 	echo Running tests ... junit_file=$(JUNIT_FILE)
 	go test ./... -v 2>&1 | go-junit-report > $(JUNIT_FILE) --set-exit-code
 
+
```
Comment (theofpa) [inline Makefile:306]: Clean-up target `test-junit` above, previously used by ksonnet.

**Review by theofpa — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (PatrickXYS) [reply Makefile]: Addressed in latest commit

**Review by theofpa — APPROVED** [review]

**Pull request merged**